src/util/cliConfiguration.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// import {
+// 	ForceConfigGet,
+// 	GlobalCliEnvironment,
+// } from '../packages/salesforcedx-utils-vscode/src/cli';
+import * as vscode from 'vscode';
+const sfdxUtilsExtension = vscode.extensions.getExtension('salesforce.salesforcedx-utils-vscode');
```

Comment:
`salesforce.salesforcedx-utils-vscode` this is actually not an extension. You can find your installed extensions from`$HOME/.vscode/extensions` if you are on macOS / Linux.

You would want to use `salesforce.salesforcedx-vscode-core`. Check out the example here: 
https://github.com/forcedotcom/salesforcedx-vscode/blob/ae50991b99f64750badb9f633da6e3c7694cbdd8/packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts#L188-L193

Reply:
Thank you!